views/helpers/text-helpers.njk

```diff
@@ -0,0 +1,13 @@
+{% macro bold(text) %}
```

Comment:
Dont think we need this helpers, you can use nunjucks filter 'safe' to tell nunjucks not to escape a text, so you could have on en.json html rather than just text

https://mozilla.github.io/nunjucks/templating.html#safe